build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -253,6 +254,34 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         }
 
         [Target]
+        public static BuildTargetResult BuildProjectsForNuGetPackages(BuildTargetContext c)
+        {
+            if (CurrentPlatform.IsWindows)
+            {
+                var configuration = c.BuildContext.Get<string>("Configuration");
+
+                // build projects for nuget packages
+                var packagingOutputDir = Path.Combine(Dirs.Intermediate, "forPackaging");
+                Mkdirp(packagingOutputDir);
+                foreach (var project in PackageTargets.ProjectsToPack)
+                {
+                    // Just build them, we'll pack later
+                    var packBuildResult = DotNetCli.Stage0.Build(
+                        "--build-base-path",
```

Comment:
(nit): it makes sense to me to put the 2 args that go together on the same line.  I think it is more readable.

```
"--build-base-path", packagingOutputDir,
"--configuration", configuration,
```